js/analyses/checkStringForStopwords.js

```diff
@@ -0,0 +1,23 @@
+/** @module analyses/checkStringForStopwords */
+
+var stopwords = require( "../config/stopwords.js" );
```

Comment:
If we do `var stopwords = require( "../config/stopwords.js" )();` here we don't have to do line 13.